src/custom-code-helpers/domain-lock/templates/DomainLockTemplate.ts

```diff
@@ -96,14 +93,13 @@ export function DomainLockTemplate (): string {
                     }
                 }
             }
-               
+
             if (!ok) {
-                data;
-            } else {
-                return;
+                const regExp2 = new RegExp("[{domainDestDiff}]", "g");
+                const domainDest = "{domainDest}".replace(regExp2, "");
+
+                that[document][location] = domainDest;
```

Comment:
How about `window.location.replace`?
With same `isName` find window->location, then location->assign, then `bind` it to location or execute with `call`

Reply:
What is wrong with `document.location = `?

Reply:
I think it's too easy to detect of attacker bypasses `stringArray` or set global `onbeforeunload` breakpoint.

Reply:
I don't think setting a breakpoint at `onbeforeunload` will allow you to see where the location was changed right?
I'm curious, how would by passing `stringArray` not work with `window.location.replace("...")` as well?